src/test/modelconfig_test.cpp

```diff
@@ -491,3 +491,183 @@ TEST(ModelConfig, ConfigParseNodeWithForbiddenShapeName) {
     ASSERT_EQ(status, ovms::StatusCode::OK);
     EXPECT_EQ(modelConfig.getShapes().size(), 0);
 }
+
+TEST(ModelConfig, ConfigParseNodeWithStatefulParams) {
```

Comment:
Rename tests to possibly describe what they do & what the end result should be:
Here it could be:
ConfigParseNodeWithStatefulParams->ParseModelLowLatencySetWithoutStatefulShouldFail
ConfigParseNodeWithStatefulParamsTransformation->ParseModelLowLatencySetWithStatefulFalseShouldFail
etc

Then you don't need to look at the test to know whats wrong.